src/test/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelperTest.java

```diff
@@ -161,4 +162,38 @@ public void getStudyInfo() {
         assertEquals(studyInfo.getDataAccessTeamId().longValue(), 1337);
         assertEquals(studyInfo.getSynapseProjectId(), "test-synapse-table");
     }
+
+    @Test
+    public void getStudyInfoNullDataAccessTeam() {
+        // mock DDB Study table - only include relevant attributes
+        Item studyItem = new Item().withString("synapseProjectId", "test-synapse-table");
+
+        Table mockStudyTable = mock(Table.class);
+        when(mockStudyTable.getItem("identifier", "test-study")).thenReturn(studyItem);
+
+        // set up Dynamo Helper
+        DynamoHelper helper = new DynamoHelper();
+        helper.setDdbStudyTable(mockStudyTable);
+
+        // execute and validate
+        StudyInfo studyInfo = helper.getStudyInfo("test-study");
+        assertNull(studyInfo);
```

Comment:
If I remember, this is null because the builder returns null if the study doesn't have these two configuration keys? Maybe a note here